samples/sensor/th02/sample.yaml

```diff
@@ -4,4 +4,4 @@ tests:
   test:
     harness: sensor
     tags: sensors
-    depends_on: i2c
+    depends_on: i2c display
```

Comment:
I'm concerned this is going to filter out the boards that this sample was designed for. I think we need to use a whitelist instead, but I'm not sure which boards to whitelist. @nashif?

I've also provided a workaround for rt1064 in #14268

Reply:
I think it is better to exclude the mimxrt1064_evk board from test and add a new entry to tests called test_mimxrt1064_evk that whitelist the mimxrt1064_evk board.